packages/design-system/src/components/Alert/Alert.test.tsx

```diff
@@ -4,19 +4,31 @@ import { UtagContainer } from '../analytics';
 import { setAlertSendsAnalytics } from '../flags';
 import { render, screen } from '@testing-library/react';
 
+import register from 'preact-custom-element';
+register(Alert, 'ds-alert');
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-alert': AlertProps;
+    }
+  }
+}
+
```

Comment:
we only need this import,`register()`, and TS interface for testing web components. tried modifying `testMatch` in jest.config and couldn't get this kind of dynamic import working...